geoportailv3/static/js/measure/measuredirective.js

```diff
@@ -47,13 +47,27 @@ app.module.directive('appMeasure', app.measureDirective);
 
 /**
  * @param {angular.Scope} $scope Scope.
+ * @param {angular.$q} $q
+ * @param {angular.$http} $http
  * @param {ngeo.DecorateInteraction} ngeoDecorateInteraction Decorate
  *     interaction service.
+ * @param {string} elevationServiceUrl the url of the service
  * @constructor
  * @export
  * @ngInject
  */
-app.MeasureController = function($scope, ngeoDecorateInteraction) {
+app.MeasureController = function($scope, $q, $http, ngeoDecorateInteraction,
+    elevationServiceUrl) {
+
+  /**
+   * @private
+   */
+  this.http_ = $http;
```

Comment:
Very minor, but we could use `this.$http_` to make it clear and explicit that it's an Angular service.